client/src/theme.js

```diff
@@ -1,58 +1,67 @@
 import { createTheme } from "@mui/material";
 
-// TODO: Add brand colors map
-// eslint-disable-next-line
-const BRAND_COLORS = {
+const COLORS = {
+  HYPERLINK_BLUE: "#005EA2",
+  BODY_TEXT: "#2F3032",
+  LIGHT_GREY_TEXT: "#78787A",
+  BACKGROUND: "#FAF0E7",
+  CARD_BG: "#FFFFFF",
+  NAVBAR_BG: "#F2DAC4",
   BLUEBERRY: "#021944",
   LYCHEE: "#F6A794",
-  MATCHA: "#B1D5B2",
-  MILK_TEA: "#F2DAC4",
-  TARO: "#E4CFF7",
+  CARD_CHIPS: "#ABBFE3",
+  ERROR_BUTTON: "#D21C1C",
+  ERROR_BG: "#F8D7D7",
+  SLIDEOUT_NOTES_BG: "#FFF5ED",
   THAI_TEA: "#E0A878",
+  MILK_TEA: "#F2DAC4",
+  LOGO_BROWN: "#98513A",
+  OFFWHITE_GREY: "#D9D9D9",
+  PAGINATION_BG: "#F9E5DC",
+  SEARCHBAR_ICON_BORDER: "#DBAE9C",
+  THAI_FILL_INACTIVE: "#B4AAA0",
+  BOBA_POSITION: "#A9917B",
 };
 
 const theme = createTheme({
   palette: {
-    brown: {
-      50: "#FAF0E7",
-      100: BRAND_COLORS.MILK_TEA,
-      200: "#E0A878",
-      300: "#A4857B",
-      400: "#98513A",
-      500: "#683323",
+    primary: {
+      main: COLORS.LYCHEE,
+      light: COLORS.PAGINATION_BG,
+      dark: COLORS.SEARCHBAR_ICON_BORDER,
```

Comment:
FYI, i want to say (might need to verify) that dark and light variants are generated if a main is provided... but don't quote me on this.

I want to deter the use of very _specific_ naming especially when coming to theme values. for example `PAGINATION_BG` is very specific to pagination. I would ask why is our primary.light variant using the same color as pagination background? Where the direction that might make more sense is

`primary.light = someNamedColor || "#someColor"`

`<pagination sx={{backgroundColor: theme.primary.light}}>`

We want to go from generic -> specific when possible.

like OFFWHITE_GREY is fine, it describes a color, but _not_ where we expect to use it. the theme and component overrides is where we can affect that kind of specification.

Reply:
I was given the palette that was used and tried to assign each colour to a part of theme that's associated as closely as possible. I've added pagination bg to primary light cuz it looks similar to primary main and i'm not where else it should be added to. I do know that dark and light variants are autogenerated but since design team already has all colours planned out I tried to add them all to the theme. Can specify what you meant by
```primary.light = someNamedColor || "#someColor"```

Reply:
so if you see the pagination bg color is similar to primary light, instead of saying `primary.light = pagination_bg`

we should do something like `<Pagination backgroundColor={theme.primary.light}>`

Reply:
so _pagination_ is using the theme's primary light variant, not the theme is using pagination background as a color. This keeps it much more generic and holding the theme palette as the source of truth for the color base

Reply:
so using the autogenerated one instead?

Reply:
I don't want to block this too long, we can merge this in and then continue with another PR to tweak? i think razi was blocked or waiting for some changes you were making here, too

Reply:
okay, that works!

Reply:
I don't have permission to merge it. Would you mind doing it?